event_model/__init__.py

```diff
@@ -1384,6 +1389,11 @@ def pack_datum_page(*datum):
     -------
     datum_page : dict
     """
+    if not datum:
+        raise ValueError(
+            "The pack_datum_page() function was called with empty *args. "
+            "Cannot create an DatumPage from an empty collection of Datum "
+            "because the 'resource' field in an DatumPage cannot be NULL.")
```

Comment:
```suggestion
            "because the 'resource' field in a DatumPage cannot be NULL.")
```

Reply:
"Datums" makes me cringe since the English (Latin, really) plural of "datum" is "data".